pkg/controller/podautoscaler/horizontal_test.go

```diff
@@ -3126,4 +4215,4 @@ func TestNoScaleDownOneMetricEmpty(t *testing.T) {
 	tc.runTest(t)
 }
 
-// TODO: add more tests
+// TODO: add more tests for stabilization
```

Comment:
Hmm ... I don't really find this so useful.  If you know of missing test cases, you could leave a commented list in the table test.

Reply:
Oh, this is an obsolete comment. I'm removing it.